package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sqs-consumer",
-  "version": "5.7.5",
+  "version": "5.7.6",
```

Comment:
after all the changes should i revert those commits? and re add the version? what should i do?